package.json

```diff
@@ -96,7 +97,7 @@
     },
     "dependencies": {
         "@fortawesome/fontawesome-free": "^5.15.2",
-        "@fortawesome/fontawesome-svg-core": "^1.2.34",
+        "@fortawesome/fontawesome-svg-core": "~1.2.34",
```

Comment:
this is fixing a spontaneous bug that popped up with weird incompatibilities due to updates in fontawesome.  I had a github link to the actual issue somewhere....